jipdate.py

```diff
@@ -229,16 +259,28 @@ def get_jira_issues(jira, username):
 
     f.write(msg)
     vprint("Found issue:")
+
+    unsorted_issues={}
     for issue in my_issues:
+        issue_id = "%s" % issue
+        this_issue = {}
+
+        this_issue["comments"] = ""
         vprint("%s : %s" % (issue, issue.fields.summary))
-        f.write("[%s]\n" % issue)
-        f.write("# Header: %s\n" % issue.fields.summary)
-        f.write("# Type: %s\n" % issue.fields.issuetype)
-        f.write("# Status: %s\n" % issue.fields.status)
-        f.write(get_extra_comments())
-        if last_comment:
-            write_last_jira_comment(f, jira, issue)
-        f.write("\n")
+        this_issue["comments"] += "[%s]\n" % issue
```

Comment:
My goal was to keep a kind of hierarchy
[parent1]
[child11]
[child12]
[parent2]
[child21]
[child22]

But when a parent (parent2) is not in the list of issues to update we have
[parent1]
[child11]
[child12]
[child21]
[child22]

Which can be a bit confusing so my goal was to add a comment when we change the parent
[parent1]
[child11]
[child12]
#[parent2]
[child21]
[child22]

Furthermore, this can be easily used to add a summary of childs activities into the parent by uncommenting the issue id